recipes-core/packagegroups/packagefeed-ni-extra.bb

```diff
@@ -302,7 +302,7 @@ RDEPENDS:${PN} += "\
 "
 
 # meta-openembedded/meta-oe/recipes-devtools
-RDEPENDS:${PN}:append:x64 += "\
+RDEPENDS:${PN}:append:x64 = "\
```

Comment:
Check instances throughout this recipe. Maybe `RDEPENDS:${PN}:x64 += ...` instead?

Reply:
From output of `bitbake -e packagefeed-ni-extra`, `RDEPENDS` contains the affected packages (verified `msr-tools`, `jasper`, `mcelog`, `puppet`, `ipxe`).